lib/miq_expression.rb

```diff
@@ -625,10 +625,14 @@ def sql_supports_atom?(exp)
       # => TODO: support count of child relationship
       return false if exp[operator].key?("count")
 
-      return field_in_sql?(exp[operator]["field"])
+      return field_in_sql?(exp[operator]["field"]) && value_in_sql?(exp[operator]["value"])
```

Comment:
@imtayadeway Do we want this in all 3 spots that `field_in_sql?` is used?

I know @alongoldboim said this will only used for integers at this time, but I'd expect it to quickly go to character fields. These features tend to take off